From 6dc69d559935a4e80f72ad92f37ddf53324c3ecc Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 28 Jan 2022 11:41:00 +0100
Subject: [PATCH] kernel: bump 4.14 to version 4.14.263

All patches refreshed without any conflicts.

Run tested on mvebu/turris-omnia

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 include/kernel-version.mk                                     | 4 ++--
 ...-netfilter-reduce-size-of-hook-entry-point-locations.patch | 2 +-
 .../644-net-pppoe-support-hardware-flow-table-offload.patch   | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index ec35588b45..d1596e6afb 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -6,9 +6,9 @@ ifdef CONFIG_TESTING_KERNEL
   KERNEL_PATCHVER:=$(KERNEL_TESTING_PATCHVER)
 endif
 
-LINUX_VERSION-4.14 = .262
+LINUX_VERSION-4.14 = .263
 
-LINUX_KERNEL_HASH-4.14.262 = 4cfd785381edbfcd86dfb1cd8d264d872643d71afd2b832fa2090f5b7228d417
+LINUX_KERNEL_HASH-4.14.263 = 0e73b74244644c03ab3455b2558131d7df4e22d573afb3f13a83cd5ac33dc12e
 
 remove_uri_prefix=$(subst git://,,$(subst http://,,$(subst https://,,$(1))))
 sanitize_uri=$(call qstrip,$(subst @,_,$(subst :,_,$(subst .,_,$(subst -,_,$(subst /,_,$(1)))))))
diff --git a/target/linux/generic/backport-4.14/293-v4.16-netfilter-reduce-size-of-hook-entry-point-locations.patch b/target/linux/generic/backport-4.14/293-v4.16-netfilter-reduce-size-of-hook-entry-point-locations.patch
index ed532a0ee0..90e3500f0e 100644
--- a/target/linux/generic/backport-4.14/293-v4.16-netfilter-reduce-size-of-hook-entry-point-locations.patch
+++ b/target/linux/generic/backport-4.14/293-v4.16-netfilter-reduce-size-of-hook-entry-point-locations.patch
@@ -92,7 +92,7 @@ Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
  #endif
 --- a/net/bridge/br_netfilter_hooks.c
 +++ b/net/bridge/br_netfilter_hooks.c
-@@ -994,7 +994,7 @@ int br_nf_hook_thresh(unsigned int hook,
+@@ -993,7 +993,7 @@ int br_nf_hook_thresh(unsigned int hook,
  	unsigned int i;
  	int ret;
  
diff --git a/target/linux/generic/pending-4.14/644-net-pppoe-support-hardware-flow-table-offload.patch b/target/linux/generic/pending-4.14/644-net-pppoe-support-hardware-flow-table-offload.patch
index c7aa998517..40a539be9d 100644
--- a/target/linux/generic/pending-4.14/644-net-pppoe-support-hardware-flow-table-offload.patch
+++ b/target/linux/generic/pending-4.14/644-net-pppoe-support-hardware-flow-table-offload.patch
@@ -21,7 +21,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  #define PPP_VERSION	"2.4.2"
  
  /*
-@@ -1393,12 +1398,37 @@ static void ppp_dev_priv_destructor(stru
+@@ -1398,12 +1403,37 @@ static void ppp_dev_priv_destructor(stru
  		ppp_destroy_interface(ppp);
  }
  
-- 
2.32.0

